be/src/olap/rowset/segment_v2/column_reader.cpp

```diff
@@ -1186,14 +1204,20 @@ void DefaultValueColumnIterator::insert_default_data(const TypeInfo* type_info,
     case OLAP_FIELD_TYPE_VARCHAR:
     case OLAP_FIELD_TYPE_CHAR:
     case OLAP_FIELD_TYPE_JSONB: {
-        data_ptr = ((Slice*)mem_value)->data;
-        data_len = ((Slice*)mem_value)->size;
+        char* data_ptr = ((Slice*)mem_value)->data;
+        size_t data_len = ((Slice*)mem_value)->size;
         dst->insert_many_data(data_ptr, data_len, n);
         break;
     }
+    case OLAP_FIELD_TYPE_ARRAY: {
+        for (int i = 0; i < n; i++) {
+            dst->insert(doris::vectorized::Array(0));
```

Comment:
why not just use dst->insert_many_default?

Reply:
Because  columnNullable will make null value as default.   So  in this function is used to process the default  value  situation.   
![image](https://user-images.githubusercontent.com/67053339/222765649-419fe6fd-24e6-419b-9958-997a41088465.png)

Reply:
> why not just use dst->insert_many_default?

Because columnNullable will make null value as default. So in this function is used to process the default value situation.
![image](https://user-images.githubusercontent.com/67053339/222766002-85ed8012-8f51-4196-a3ea-ca015135d237.png)

Reply:
you could get the nested type of ColumnNullable and call `insert_many_default` and assign the null map of ColumnNullable with may `0`, the could be much faster

Reply:
> you could get the nested type of ColumnNullable and call `insert_many_default` and assign the null map of ColumnNullable with may `0`, the could be much faster

ok